src/main/java/com/sda/auction/controllers/AddressController.java

```diff
@@ -0,0 +1,14 @@
+package com.sda.auction.controllers;
+
+import com.sda.auction.services.AddressService;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
Since we use lombok, we can add @AllArgsConstructor(onConstructor = @__({Autowired}))
With this, you will not need to add an explicit constructor and it will inject all your dependencies

Reply:
Same applies for all